app/ide-desktop/lib/dashboard/public/index.html

```diff
@@ -0,0 +1 @@
+../../content/src/index.html
```

Comment:
1. What is this file for?
2. As I've written in previous PR and we've been discussing on calls, the `content` should be served both in the desktop app and in the cloud, so the `lib/dashboard` should be only a dependency of content and should not be build as a separate package for cloud. This file suggests me that this is not the case, am I correct?

Reply:
Yes we agreed on that and it will be that way.

For now we are focusing on delivering working authorisation for both IDE and cloud. Currently for development and testing purpose we run cloud with `npm run dev` see https://github.com/enso-org/enso/pull/5798/files#diff-300d31ce155ef94319a2038cbae42f31f9e60ef6a3c07adae55ea4630ed1365dR37

React scripts require presence of `public/index.html` in app root. This symlink which points to index.html from content directory is a temporary solution to keep only 1 version of this file. 

We will refactor this in PR that creates proper entrypoint for dashboard

Reply:
This file is a symlink, not a regular file. So we can't add a FIXME to it. I will add a FIXME at the target file though.